2.4/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -244,7 +244,15 @@ function setup_keyfile() {
     exit 0
   fi
   if [ -z "${MONGODB_KEYFILE_VALUE-}" ]; then
-    echo "ERROR: You have to provide the 'keyfile' value in MONGODB_KEYFILE_VALUE"
+    echo >&2 "ERROR: You have to provide the 'keyfile' value in MONGODB_KEYFILE_VALUE"
+    exit 1
+  fi
+  MONGODB_KEYFILE_DIR="$(dirname "$MONGODB_KEYFILE_PATH")"
+  if [ ! -w "$MONGODB_KEYFILE_DIR" ]; then
+    echo >&2 "ERROR: Couldn't create ${MONGODB_KEYFILE_PATH}"
+    echo >&2 "CAUSE: current user doesn't have permissions for writing to ${MONGODB_KEYFILE_DIR} directory"
```

Comment:
There are many causes that can lead to this error and I don't think that it's possible to cover all of them. For example, SELinux problems not covered here (actually, I tried to use `id -Z` and `stat -c %C` but `id -Z` shows that there is no selinux context inside of container).